packages/cli/src/controllers/mfa.controller.ts

```diff
@@ -86,13 +86,22 @@ export class MFAController {
 	@Post('/disable', { rateLimit: true })
 	async disableMFA(req: MFA.Disable) {
 		const { id: userId } = req.user;
-		const { mfaCode = null } = req.body;
 
-		if (typeof mfaCode !== 'string' || !mfaCode) {
-			throw new BadRequestError('Token is required to disable MFA feature');
+		const { mfaCode, mfaRecoveryCode } = req.body;
+
+		const mfaCodeDefined = mfaCode && typeof mfaCode === 'string';
+
+		const mfaRecoveryCodeDefined = mfaRecoveryCode && typeof mfaRecoveryCode === 'string';
+
+		if (!mfaCodeDefined && !mfaRecoveryCodeDefined) {
+			throw new BadRequestError('MFA code or recovery code is required to disable MFA feature');
```

Comment:
Maybe we should allow only one or the other to be defined:

```suggestion
		if (!!mfaCodeDefined == !!mfaRecoveryCodeDefined) {
			throw new BadRequestError('Either MFA code or recovery code is required to disable MFA feature');
```

Otherwise it isn't very clear which one is used